src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+
+
+  &__img {
+    display: block;
+    width: 160px;
+    min-height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    color: $text-color-main;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+
+    color: $text-color-secondary;
```

Comment:
in block like this (where you have a few style rules) you can put them together without adding a line between them